.cirrus.default.yml

```diff
@@ -242,8 +240,151 @@ qa_task:
       /etc/init.d/xvfb stop
     test_recording_artifacts:
       path: "${CIRRUS_WORKING_DIR}/recording_${QA_CATEGORY}.mp4"
+    jacoco_it_sq_artifacts:
+      path: "${CIRRUS_WORKING_DIR}/it-coverage*.exec"
+    mvn_startTestLog_artifacts:
+      path: "${CIRRUS_WORKING_DIR}/its/**/target/work/configuration/*.log"
+    mvn_runTestLog_artifacts:
+      path: "${CIRRUS_WORKING_DIR}/its/**/target/work/data/.metadata/.log"
+  on_failure:
+    xvfb_log_artifacts:
+      path: "${CIRRUS_WORKING_DIR}/Xvfb.out"
+    reports_artifacts:
+      path: "**/reports/**/*"
+    junit_artifacts:
+      paths:
+      - "**/test-results/**/*.xml"
+      - "**/target/surefire-reports/TEST-*.xml"
+      format: junit
+    slack_notification_script: |
+      source slack-failure-notification "Cirrus CI build failure on pipeline: $CIRRUS_PIPELINE_NAME"
+
+#qa_connectedModeSonarCloud_task:
```

Comment:
I wouldn't keep this as commented, even if it's not yet enabled. Perhaps you could use `skip` or `only_if` and directly set the boolean

Reply:
I will immediately start working on this, that's why it is still in here :smile: